app/services/persister/transfer_persister.rb

```diff
@@ -1,9 +1,10 @@
 module Persister
   class TransferPersister
-    attr_accessor :transfer
+    attr_accessor :transfer, :organization
 
-    def initialize(transfer)
+    def initialize(transfer, organization)
       @transfer = transfer
+      @organization = organization
```

Comment:
Can't we infer from `@transfer.organization`?

Reply:
transfer doesn't have org id, It should be something like transfer.post.organization but then I realised that it was easier to pass it as an option to the Persister.

Reply:
The less we expose the better, I vote for using `transfer.post.organization`